scripts/generate-openapi.js

```diff
@@ -0,0 +1,17 @@
+import { writeFile } from 'node:fs/promises'
+
+const generateOpenapi = async (path = './src/docs/openapi/v1.json') => {
+  try {
+    const openApiResponse = await fetch('http://localhost:3004/documentation.json')
```

Comment:
It would have been nice to get this port out of the configuration, but that is not strictly needed, so just a comment to ignore :)

Reply:
I agree, could get it from `.env` but we're not using a `.env` for anything else in this service as we pull config from docker-compose when running the service.

But as this script isn't running via Docker that wont work... As its a standalone script I think hardcoding for now will be sufficient.